pkg/cmd/infra/deployer/deployer.go

```diff
@@ -78,7 +78,7 @@ func NewCommandDeployer(name string) *cobra.Command {
 }
 
 // NewDeployer makes a new Deployer from a kube client.
-func NewDeployer(client kclient.Interface) *Deployer {
+func NewDeployer(client *kclient.Client) *Deployer {
```

Comment:
As I mention in the description and linked issue: I can't find any other way to get access to the pod log stream. None of the interfaces expose anything helpful for it that I can find. Suggestions?

Also, I don't necessarily think using the concrete client for initialization code (constructors) called only by other initialization code is a bad thing.